ingest/README.md

```diff
@@ -44,6 +44,16 @@ nextstrain build \
     . upload_all
 ```
 
+### Ingest from Andersen lab's avian-influenza repo
+
+Ingest publicly available consensus sequences and metadata from Andersen lab's [avian-influenza repo](https://github.com/andersen-lab/avian-influenza).
+
+```sh
+nextstrain build . merge_andersen_segment_metadata
+```
+
+The results will be available in `results/andersen-lab/`.
```

Comment:
I [was expecting](https://github.com/nextstrain/avian-flu/issues/26#issue-2278239875) this PR to merge/dedup the data with that coming from fauna:

> 9. Concatenate fauna and Andersen lab files
> 10. Dedup based on strain name
> 11. Upload merged data to S3

However since the currently available data is now available via Fauna I can see [you noted](https://github.com/nextstrain/avian-flu/issues/26#issuecomment-2096678566):

> holding off on uploading Andersen lab data to S3 and merging with fauna data since the current sequences have become available via GISAID... Will make a PR to add the Andersen lab workflow to ingest/ in case we need to ingest any new data in the future.

Is the plan to keep this PR unmerged and revisit it when new data arrives (adding the merge/dedup at that point), or would you like to merge now and add the merge/dedup steps if/when the need arises?

Reply:
I was planning to merge now-ish and add merge/dedup steps later when the need arises. 
I can add more notes in the README to clarify if/when to use this part of the ingest workflow.

Reply:
Added more details in [force-push](https://github.com/nextstrain/avian-flu/compare/5bfb10a8f5063413502c1ac578cab14a1a09131b..b493d496d4f239db81bd8c14baad5f52ac8afc68)